demo/kubernetes/rook-operator.yaml

```diff
@@ -12,5 +12,9 @@ spec:
     spec:
       containers:
       - name: rook-operator
-        image: quay.io/rook/rook-operator:dev-2017-02-17-k8s
-        args: ["--container-version=dev-2017-02-17-k8s"]
+        image: quay.io/rook/rook-operator:dev-2017-02-23-k8s
```

Comment:
I think we shouldnt be chaning the image tag. It should just be `image: quay.io/rook/rook-operator`. When developing, we can specify the tag. When we merge, we should publish a new 'latest' image.